glos_qartod/qc.py

```diff
@@ -52,9 +85,20 @@ def find_ancillary_variables(self, ncvariable):
         :param netCDF4.Variable ncvariable: Variable to get the ancillary
                                             variables for
         '''
+
+        missing_var_template = """<variable name="{}">
```

Comment:
Could you move this template to the top of the file so that the template variables are kept together, not to mention it doesn't look very good to have this multi-line string in the middle of the method like this.